docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -86,19 +85,25 @@ For more details, see the Kubernetes documentation about [Creating a new namespa
    helm repo update
    ```
 
-   > **NOTE**: The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+   {{% admonition type="note" %}} The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+   {{% /admonition %}}
 
 ## Set Helm chart values
 
-Your Helm chart values are set in the `custom.yaml` file. The following example `custom.yaml` file sets the storage and traces options, enables the gateway, and sets the cluster to main. The `traces` configure the distributor's receiver protocols.
+Your Helm chart values are set in the `custom.yaml` file.
+The Tempo Helm char values example `custom.yaml` file sets the storage and traces options, enables the gateway, and sets the cluster to main.
+The `traces` configure the distributor's receiver protocols.
 
-Next, you will:
+To customize your Helm chart values:
 
-1. Create a `custom.yaml` file
-2. Set your storage values, the example above points to the MinIO instance configured by the chart
-3. Set your traces values to configure the receivers on the Tempo distributor
+1. Create a `custom.yaml` file.
+1. Copy and paste either the Tempo Helm chart values or the GET Helm chart values into your file.
+1. Save the file in your working directory.
+1. Set your storage values, the example helm chart values points to the MinIO instance configured by the chart.
+1. Set your traces values to configure the receivers on the Tempo distributor.
+1. Save the changes to your file.
 
-### Tempo helm chart values
+### Tempo Helm chart values
 
 This sample file contains example values for installing Tempo using Helm.
 
```

Comment:
@zachfi  Question about helm configuration for MinIO. Tempo Helm chart has values that aren't configured for GET Helm chart. Do we need to copy the MinIO section from the Tempo chart values sample to the GET Helm chart value sample? 

The Tempo Helm chart has these values for MinIO (lines 116-130): 
```
minio:
  enabled: true
  mode: standalone
  rootUser: grafana-tempo
  rootPassword: supersecret
  buckets:
    # Default Tempo storage bucket.
    - name: tempo-traces
      policy: none
      purge: false
```

GET Helm chart values (lines 160-1): 
```
minio:
  enabled: true
```

Reply:
Can you link where you see?

Reply:
GET values: https://github.com/grafana/tempo/pull/2590/files#r1244287106
Tempo values: https://github.com/grafana/tempo/pull/2590/files#r1244288148

Reply:
Oh sorry, I see now you mean in the docs not in the chart.  Yes the same logic applies.  If users want to use minio, with non-default credentials, they will need to set them in the `minio` block as well as configured for the `s3` storage.

Reply:
I'll add this to the PR.

Reply:
Okay cool.  Let me know when you're ready for a review.

Reply:
@zalegrala updated. Review?

Reply:
Hello, sorry to barge into the convo like that. I hope it is okay for me to ask a short question regarding the docs here...

I ran `helm install` with the suggested custom values from the doc. However, multiple pods like the tempo-ingester fail to start up with the logs:

> level=info ts=2023-07-13T20:05:32.435682845Z caller=main.go:215 msg="initialising OpenTracing tracer"
> level=info ts=2023-07-13T20:05:32.450606561Z caller=main.go:102 msg="Starting Tempo" version="(version=2.1.1, branch=HEAD, revision=4157d7620)"
> level=error ts=2023-07-13T20:05:32.452641937Z caller=main.go:105 msg="error running Tempo" err="failed to init module services error initialising module: store: failed to create store unexpected error from ListObjects on tempo-traces: The Access Key Id you provided does not exist in our records."

Am I misunderstanding the docs? From what I see on my cluster, the actual access key and secret key which I can use to log in to MinIO are base64 encoded in the secret `tempo-minio` and are not equal to the strings `grafana-tempo` and `supersecret` specified in `storage.s3.access_key` and `storage.s3.secret_key`.

Reply:
@ViviLearns2Code I think there was a mistake in the chart dependency version.  https://github.com/grafana/helm-charts/pull/2509 should correct the dependency version.  I'll test with @danstadler-pdx this afternoon.

Reply:
@knylander-grafana For the two minio examples, one includes the root credentials and one does not.  If we don't include the root credentials as an example, folks might be confused how to change those credentials, but since the values there are default then we don't need to include them.  The only required key is the `minio.enabled: true` and then default values will be used.

Reply:
@zalegrala The example was updated to include the credentials (see [section below](https://github.com/grafana/tempo/pull/2590/files#r1265465821)